src/test/enhetstester/kotlin/no/nav/familie/ks/sak/kjerne/beregning/BeregningServiceTest.kt

```diff
@@ -0,0 +1,86 @@
+package no.nav.familie.ks.sak.kjerne.beregning
+
+import io.mockk.every
+import io.mockk.impl.annotations.InjectMockKs
+import io.mockk.impl.annotations.MockK
+import io.mockk.junit5.MockKExtension
+import no.nav.familie.ks.sak.data.lagAndelTilkjentYtelse
+import no.nav.familie.ks.sak.data.lagBehandling
+import no.nav.familie.ks.sak.data.lagInitieltTilkjentYtelse
+import no.nav.familie.ks.sak.data.lagPersonopplysningGrunnlag
+import no.nav.familie.ks.sak.data.randomAktør
+import no.nav.familie.ks.sak.data.randomFnr
+import no.nav.familie.ks.sak.kjerne.behandling.domene.BehandlingÅrsak
+import no.nav.familie.ks.sak.kjerne.beregning.domene.AndelTilkjentYtelseRepository
+import no.nav.familie.ks.sak.kjerne.personopplysninggrunnlag.domene.PersonopplysningGrunnlagRepository
+import org.junit.jupiter.api.Assertions.assertTrue
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.extension.ExtendWith
+
+@ExtendWith(MockKExtension::class)
+class BeregningServiceTest {
+
+    @MockK
+    private lateinit var andelTilkjentYtelseRepository: AndelTilkjentYtelseRepository
+
+    @MockK
+    private lateinit var personopplysningGrunnlagRepository: PersonopplysningGrunnlagRepository
+
+    @InjectMockKs
+    private lateinit var beregningService: BeregningService
+
+    @Test
+    fun `finnBarnFraBehandlingMedTilkjentYtelse skal ikke returnere når det ikke finnes en andel tilkjent ytelse`() {
```

Comment:
```suggestion
    fun `finnBarnFraBehandlingMedTilkjentYtelse skal returnere når tom liste når det ikke finnes en andel tilkjent ytelse`() {
```
Denne metoden returnerer en tom liste hvis det ikke finnes andel tilkjent ytelse, så det er litt mer riktig å bytte til det her